alot/commands/__init__.py

```diff
@@ -41,6 +55,23 @@ class CommandCanceled(Exception):
     'global': {},
 }
 
+# classes as keys with (mode, cmdname) as values
+REVERSE_COMMANDS = {}
+
+
+def reverse_lookup_command(klass):
+    """
+    returns mode and commandname for a commandclass
+
+    :param klass: the class used to look up the cmdname and mode
+    :type klass: class
+    :rtype: (str, str)
+    """
+    if klass in REVERSE_COMMANDS:
+        return REVERSE_COMMANDS[klass]
+    else:
+        return (None, None)
```

Comment:
please use `class_` instead of `klass` for the name, this is in keeping with PEP8 standards:
```
* single_trailing_underscore_ : used by convention to avoid conflicts with Python keyword, e.g.

Tkinter.Toplevel(master, class_='ClassName')
```

You can also simplify this function by using `dict.get()` instead:
```python
return REVERSE_COMMANDS.get(class_, (None, None))
```